predictsignauxfaibles/evaluate.py

```diff
@@ -128,24 +137,39 @@ def evaluate(
             If provided, the model will classify an entry X as positive if predict_proba(X)>=thresh.
             Otherwise, the model classifies X as positive if predict(X)=1, ie predict_proba(X)>=0.5
     """
+    aucpr = average_precision_score(
+        dataset.data["outcome"],
+        model.predict_proba(dataset.data)[:, 1],
```

Comment:
I would suggest that you compute `model.predict_proba(dataset.data)[:, 1]` once and for all at the beginning of your function, store it in a variable, and use the variable every time you need after that :)

This will save you (somewhat expensive) calls of the predict function

Reply:
Yep, good idea!

Reply:
Fixed in https://github.com/signaux-faibles/predictsignauxfaibles/pull/71/commits/4d423c9dc2736296841b82dba21a0160d6844f23